src/test/java/com/linkedin/kafka/clients/largemessage/Version0DeserializerTest.java

```diff
@@ -0,0 +1,57 @@
+package com.linkedin.kafka.clients.largemessage;
+
+import com.linkedin.kafka.clients.headers.DefaultHeaderDeserializer;
+import com.linkedin.kafka.clients.headers.HeaderDeserializer.DeserializeResult;
+import com.linkedin.kafka.clients.headers.HeaderUtils;
+import java.nio.ByteBuffer;
+import java.util.Arrays;
+import java.util.UUID;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+
+
+/**
+ * Tests that we can deserialize old encodings.
+ */
+public class Version0DeserializerTest {
+
+
+  @Test
+  public void testDeserializeVersion0Headers() {
```

Comment:
Should we test version 1 message format to ensure the combined deserializer works with the new format as well?